internal/server/network/driver_common.go

```diff
@@ -1142,10 +1142,56 @@ func (n *common) getExternalSubnetInUse(ctx context.Context, tx *db.ClusterTx, u
 	}
 
 	// Get all network load balancer listen addresses for all networks (of any type) connected to our uplink.
-	projectNetworksLoadBalancersOnUplink, err = tx.GetProjectNetworkLoadBalancerListenAddressesByUplink(ctx, uplinkNetworkName, memberSpecific)
+	// all network names for all project names
+	networksByProjects, err := tx.GetNetworksAllProjects(ctx)
 	if err != nil {
-		return nil, fmt.Errorf("Failed loading network forward listen addresses: %w", err)
+		return nil, fmt.Errorf("Failed loading network load balancer listen addresses: %w", err)
```

Comment:
I'll be using this logic for retrieving the network forward listen addresses too, so maybe we can have some of these `return nil, err`?